log/log.go

```diff
@@ -54,11 +54,9 @@ type StdLogger interface {
 	Printf(format string, args ...interface{})
 	Print(...interface{})
 	Println(...interface{})
```

Comment:
The `StdLogger` has the same interface as the `stdlog.Log` struct. But that is quite the stretch if someone only looks at our types :laughing:

Reply:
I was assuming something like that. Could you extend the function docu respectively?

Reply:
I dont understand why the linters dont fail here as well, since it are undocumented interface functions :nauseated_face:  
Created issue https://github.com/hyperledger-labs/go-perun/issues/262